src/app/shared/guards/confirm-user.guard.ts

```diff
@@ -43,7 +43,14 @@ export class ConfirmUserGuard implements CanActivate {
         .subscribe((tokenObj) => {
             if (tokenObj) {
                 this.tokenService.storeToken(tokenObj);
-                this.router.navigateByUrl('/profile');
+                if(localStorage.getItem('privateConsultationId')){
```

Comment:
@Harshwardhan18 This is not necessary. The redirection will happen once the user clicks on the email verification link sent to his email. Here's the flow- 
1. I signup
2. I fill in the additional details
3. You send these details to the API along with a callbackUrl
4. The Verify your email screen opens up.
5. I go to my email and click on the verification link. My email is verified and I'm redirected back to the Private Consultation page.

Reply:
@Harshwardhan18 As mentioned, for login flow, you can use this

Reply:
@Harshwardhan18 Remove this

Reply:
@avikt already removed in new commits